indexing-service/src/main/java/io/druid/indexing/overlord/http/TaskPayloadResponse.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Licensed to Metamarkets Group Inc. (Metamarkets) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. Metamarkets licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.druid.indexing.overlord.http;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.druid.indexing.common.task.Task;
+
+import java.util.Objects;
+
+public class TaskPayloadResponse
+{
+  private final String task;
```

Comment:
`taskId` sounds more intuitive.

Reply:
I called it `task` because that's what it is in the JSON API. If we change this to `taskId` here then we will still need `@JsonProperty("task")` on the getter and constructor. I thought that wouldn't be worth it.

Reply:
What do you think?

Reply:
Oh, do you mean this is for backward compatibility? That makes sense. Then, please leave a simple comment like ```private final String task; // taskId```

Reply:
Updated the comment.